src/Server/EventBusExplorer.Server.Infrastructure.RabbitMQ/RabbitMQTopicsService.cs

```diff
@@ -28,24 +29,29 @@ public async Task<string> CreateTopicAsync(string? name, CancellationToken cance
 
     public async Task<IList<string>> GetTopicsAsync(CancellationToken cancellationToken = default)
     {
-        IList<ExchangeTopic> topics = await _adminClient.GetTopicsAsync(
+        IList<Exchange> topicExchanges = await _adminClient.GetExchangesAsync(
+            type: ExchangeType.Topic,
             cancellationToken: cancellationToken);
 
-        return topics.Select(x => x.Name).ToList();
+        return topicExchanges.Select(x => x.Name).ToList();
     }
 
     public async Task<string> GetTopicAsync(string name, CancellationToken cancellationToken = default)
     {
-        ExchangeTopic topic = await _adminClient.GetTopicAsync(
+        Exchange? topic = await _adminClient.GetExchangeAsync(
             name,
+            type: ExchangeType.Topic,
             cancellationToken: cancellationToken);
 
+        if (topic is null)
```

Comment:
@Krusty93 probably in this case we want to return at API level something like 204 or 404. 
What do you think about?